src/test/java/pages/Employment_Status.java

```diff
@@ -0,0 +1,164 @@
+            click(By.xpath(ESSaveData));
+
+            isElementVisible(By.cssSelector(ESTable));
+
+            for (WebElement EmpStatus : ESlist) {
+                String txtEmpStatus = EmpStatus.getText();
+                System.out.println(txtEmpStatus);
+
+            }
+            Boolean match = ESlist.stream().map(s -> s.getText()).anyMatch(s -> s.equalsIgnoreCase(eStatus));
+
+
+
+        }
+        return true;
+
+    }
+
+    public String success() {
+
+        return getText(By.xpath(ESSave));
+    }
+
+    public boolean EditEmpStatus() {
+        sleep(3000);
+        click(By.xpath(ESEdit));
+        /*List<WebElement> ESL = driver.findElements(By.xpath(ES));
+        for(int i = 0; i< ESL.size();i++) {
+            WebElement Edit = ESL.get(i);
+            List<WebElement> EB = driver.findElements(By.xpath(EditButton));
+            for (int j = 1; j < EB.size(); j++) {
+                //WebElement xy = EB.get(j);
+                String EH = Edit.getText();
+                if (EH.equalsIgnoreCase(editStatus)) {
+                    click(By.xpath(EditButton+"["+j+"]"));
+                    sleep(3000);
+                    */
+        sleep(5000);
+        WebElement clear = driver.findElement(By.xpath(ESEditStatus1));
+        clear.sendKeys(Keys.CONTROL + "a");
+        clear.sendKeys(Keys.BACK_SPACE);
+        /*sleep(5000);
+        isElementClickable(By.xpath(ESEditStatus1));
+        setText(By.xpath(ESEditStatus1), updatedStatus);
+        sleep(5000);
+        click(By.xpath(ESSaveData));
+
+         */
+
+        //Boolean match = ESlist.stream().map(s -> s.getText()).anyMatch(s -> s.equalsIgnoreCase(editStatus));
+
+        return true;
+
+    }
+
+    public boolean deleteEmpStatus(String deleteStatus) {
+        sleep(2000);
+        List<WebElement> ESL = driver.findElements(By.xpath(ES));
+        for(int i = 0; i< ESL.size();i++){
+            WebElement Trash = ESL.get(i);
+            List<WebElement> TB = driver.findElements(By.xpath(TrashButton));
+            for(int j=0; j<TB.size();j++) {
+                //WebElement xy = TB.get(j);
+                String EX = Trash.getText();
+                if (EX.equalsIgnoreCase(deleteStatus)) {
+                    click(By.xpath(TrashButton));
+                    sleep(2000);
+                    click(By.xpath(ESDeleteYesDelete));
+                    break;
+                }
+            }
+
+
+        }
+        return true;
+    }
+
+
+    public String deleteRandomMultipleEmpStatus() {
+        sleep(3000);
+        click(By.xpath(ESMultiDelete1));
+        click(By.xpath(ESMultiDelete2));
+        click(By.xpath(ESDeleteSelected));
+        click(By.xpath(ESDeleteYesDelete));
+        isElementVisible(By.cssSelector(ESTable));
+        System.out.println(ESTable);
+        for (WebElement EmpStatus : ESlist) {
+            String txtEmpStatus = EmpStatus.getText();
+            System.out.println(txtEmpStatus);
+
+        }
+        return null;
```

Comment:
Why this method is returning null?

Add employment statuses which you're going to delete as method parameters and make sure only those are getting deleted from table.